.github/workflows/test.yml

```diff
@@ -438,3 +470,69 @@ jobs:
   #         pixi-version: 0.1.0
   #     - run: exit 1
   #       if: success()
+
+  # test-frozen-and-locked:
+  #   runs-on: ${{ matrix.os }}
+  #   strategy:
+  #     fail-fast: false
+  #     matrix:
+  #       os: [windows-latest, ubuntu-latest, macos-latest]
+  #   steps:
+  #     - uses: actions/checkout@v4
+  #     - name: Move pixi files
+  #       run: mv test/default/* .
+  #     - uses: ./
+  #       with:
+  #         pixi-version: v0.5.0
+  #         frozen: true
+  #         locked: true
+  #     - run: exit 1
+  #       if: success()
```

Comment:
Did you mean to test this one to?
I assume this should work.

Reply:
These tests are commented out because they are failing (and are expected to fail). In GHA, you unfortunately cannot "catch" failing tests.